.github/workflows/auto_comment_pr_merge.yml

```
@@ -0,0 +1,36 @@
+  pull-requests: write
+
+jobs:
+  comment:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    if: github.event.pull_request.merged == true
+
+    steps:
+    - name: Checkout Repository
+      uses: actions/checkout@v2
```

actionlint

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)